lib/worldwide_ab_test_helper.rb

```diff
@@ -0,0 +1,57 @@
+class WorldwideAbTestHelper
+  attr_reader :content
+
+  def initialize
+    @content = YAML.load_file(file_path)
+  end
+
+  def has_content_for?(location_slug)
+    !!content_for(location_slug)
+  end
+
+  def content_for(location_slug)
+    content[location_slug]
+  end
+
+  def location_for(worldwide_organisation)
+    hard_coded_location_for(worldwide_organisation) ||
+      worldwide_organisation.world_locations.first
+  end
+
+  def is_under_test?(worldwide_organisation)
```

Comment:
Wondering if the naming of this method should be more descriptive, since since other things apart from  worldwide_organisations can be under test too (eg world locations). But I guess we can adapt the method names as we go.